setup.py

```diff
@@ -9,10 +9,10 @@
     changelog = changelog_file.read()
 
 install_requires = [
-    'django-model-utils>=4.2,<4.3',
+    'django-model-utils>=4.2,<4.4',
     'django-recaptcha>=3.0,<3.1',
-    'wagtail>=4.0,<4.1',
-    'django>=3.2,<4.2'
+    'wagtail>=4.0,<4.3',
+    'django>=3.2,<5'
```

Comment:
This will be an issue when Django 4.2 is released, right? Since Wagtail 4.2 will at that moment not yet support Django 4.2.

Reply:
not really, wagtail also limits Django versions so pip won't install that. However, due to the forward compatibility changes I've made, I expect that the full Django 4.x range should be supported by this package.